src/templates/expressions/typescriptfieldtypeconverter.ts

```diff
@@ -0,0 +1,108 @@
+const KintoneStringFieldTypeAndTypeScriptFieldTypeName = {
+    SINGLE_LINE_TEXT: "kintone.types.fields.SingleLineText",
+    MULTI_LINE_TEXT: "kintone.types.fields.MultiLineText",
+    RICH_TEXT: "kintone.types.fields.RichText",
+    DATE: "kintone.types.fields.Date",
+    NUMBER: "kintone.types.fields.Number",
+    DATETIME: "kintone.types.fields.DateTime",
+    TIME: "kintone.types.fields.Time",
+    DROP_DOWN: "kintone.types.fields.DropDown",
+    LINK: "kintone.types.fields.Link",
+    RADIO_BUTTON: "kintone.types.fields.RadioButton",
+};
+
+function convertStringField(typeName: string) {
```

Comment:
Do you have any reasons to group field types ( `KintoneStringFieldType...`, `KintoneStringListFieldType...`, etc.) ?
I thought they could be combined into one object, then we need to define just one `convert` function.

Reply:
I agree with you. I fixed to simplify converter which convert from kintone field type to typescript defined field type.